topics/transcriptomics/tutorials/pinpoint-key-pathways-with-heinz/tutorial.md

```diff
@@ -0,0 +1,385 @@
+---
+layout: tutorial_hands_on
+
+title: "Metaproteomics: Pinpoint key pathways using Heinz"
+zenodo_link: "https://doi.org/10.5281/zenodo.1344105"
+questions:
+    - "Which pathways are potentially contributing to dental caries?"
+objectives:
+    - "Analyze metatranscriptomics data using Heinz in Galaxy to pinpoint the optimal scoring subnetwork."
+time_estimation: "1-2h"
+key_points:
+    - "Analyzing differential expression for (meta)transcriptomics data."
+    - "Validating the p-value distribution of the differential expression analysis."
+    - "Finding the most differentially expressed network in a gene functional network using Heinz."
+    - "Analyzing the pinpointed network."
+contributors:
+    - cicozhang
+    - sanneabeln
+---
+
+# Overview
+{:.no_toc}
+
+In this tutorial, we will perform a network analysis using [Heinz](https://github.com/ls-cwi/heinz) in Galaxy. The data comes from the study [May et al.](https://academic.oup.com/bioinformatics/article/32/11/1678/2240171), and we will reproduce some of the computational steps from this study with simplified data and parameters to speed up the analysis for the purposes of this tutorial.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+> ### {% icon comment %} Note
+> Your results may be slightly different from the ones presented in this tutorial due to the tool versions or
+> reference data versions or stochastic processes in the algorithms.
+{: .comment}
+
+In this tutorial, we will create a Heinz workflow step by step, as the picture below shows.
+
+![Heinz workflow](../../images/heinz-workflow.png)
```

Comment:
Done.

Reply:
For the workflow, I just did the screenshot.  For the flowchart, I used XMind. Hope this helps.

Reply:
I think XMind is not Free/Open Source (not even freeware) though.

Reply:
@nsoranzo That's not my perspective actually. I am a staunch supporter of open source; however good stuff is not equivalent with open source/free, every product has its own way to survive. However, Xmind seems to be open source, https://github.com/xmindltd/xmind, and of course I pay for it because it is better/cheaper than the alternatives from my perspective, I pay to support this powerful tool, making my logic thinking much easier. Open source does not mean it must be free, doesn't it? By the way, Xmind has a free version, which suffices for most activities and I had been a free user for many years (7+).
They always support Linux, that's nice. :smile:

Reply:
Ah, good to know there's also an open source version, I looked a bit on their website https://www.xmind.net/ and found no mention of it, now I see something in the FAQ.

> Open source does not mean it must be free, doesn't it?

No, but it's not the open source version that you pay for here, but for the Zen or Pro versions which have extra features. Before recommending Xmind as the choice for creating graphics for Galaxy training, someone needs to check that the open source version is good enough (I'm not an expert at all).

Reply:
just FYI, see also [this PR](https://github.com/galaxyproject/training-material/pull/129) using yED to make an image like the metagenomics one I linked to in comment above.

the template file seems to have been moved around a bit since that PR but a copy is still [here](https://github.com/galaxyproject/training-material/blob/master/topics/metagenomics/images/template.graphml) (we should move that somewhere more prominent)

Reply:
@nsoranzo I am not recommending anything, I am just sharing my experience, though it is not easy to differentiate it from the texts (I will make it more explicit next time). :smile:

Reply:
There's also [draw.io](https://www.draw.io/), available online and on desktop (Win/Linux/macOS/Chrome), which is open source. No idea how it compares in terms of features.